assets/data-labeling/environments/data-labeling/context/conda_dependencies.yaml

```diff
@@ -15,4 +15,6 @@ dependencies:
   - pycairo==1.20.1
   - pydicom=={{latest-pypi-version}}
   - python-rle==0.0.3
+  - cryptography=={{latest-pypi-version}}
```

Comment:
I don't think you need to pin/add cryptography or certifi. Those should be unpinned nested references from the packages you depend on.

Reply:
Okay, I removed them, do we just need to create a new build with AzureML-Assets-Environments Pipeline and release?